PROVESFlightControllerReference/test/int/common.py

```
@@ -98,6 +127,15 @@ def proves_send_and_assert_command(
                 and (attempt + 1) % RADIO_RECOVER_THRESHOLD == 0
             ):
                 _radio_recover_fn()
+            # A mid-test reboot (safe-mode entry, reset, watchdog) leaves the
+            # board expecting a written-ahead sequence number (issue #461) and
+            # silently rejecting every authenticated command. GET_SEQ_NUM is
+            # bypass-listed, so resyncing here works even in that state and
+            # costs one round-trip per failed attempt.
+            try:
+                resync_sequence_number(fprime_test_api)
+            except Exception:  # noqa: BLE001 -- recovery must not mask the retry
+                pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)